public/app/plugins/panel/canvas/editor/QuickPositioning.tsx

```diff
@@ -0,0 +1,112 @@
+import { css } from '@emotion/css';
```

Comment:
This is looking / working great for elements in top vertical constraint and left horizontal constraint. For all other constraints this unfortunately seems to be broken :/ 

I can look into this tomorrow to address the other constraint options 👍

Reply:
Took a look into this. After some trial and error, seems easiest solution appears to be converting the element's constraints to be vertical - top, horizontal - left, applying the quick positioning, and then re-updating the elements constraints back to their original value. While this approach works / UX wise seems ok, longer term this may not be the best solution but perhaps "good enough" for the time being

@ryantxu thoughts on this approach? If ok, I'm good to approve / merge this 👍 

https://user-images.githubusercontent.com/22381771/169628198-a1c36c68-b629-49c9-9015-c519bd0907d8.mov

Reply:
> appears to be converting the element's constraints to be vertical - top, horizontal

This seems like a reasonable approach!  We may want to do the same thing for "drag start" / "stop" -- given the complexity of all the variations, this may be the easiest way to avoid "jumping"